### ISSUE TITLE
Add timestamps

### ISSUE THREAD

**Issue by jtojnar** [Push-EDX/flarum-ext-chat#26]
The ability to see when was a message sent would be quite useful, even as just a tooltip (though mobile users would be left in the cold)

**Comment by jtojnar** [comment]
#29 will start saving the timestamps.

**Comment by gpascualg** [comment]
Indeed, it's been on my head for ages but have never had the time to actually write it.

I've already merged that pull request, so now it is a question of simply showing it. If I get some time I'll do it myself
